cmd/mtc/main.go

```diff
@@ -174,20 +175,16 @@ func assertionFromFlagsUnchecked(cc *cli.Context) (*mtc.AssertionRequest, error)
 			}
 		}
 
-		var a mtc.Assertion
-		err = a.UnmarshalBinary(assertionBuf)
+		var ar mtc.AssertionRequest
```

Comment:
bugfix: previously trying to parse an Assertion but should have been an AssertionRequest.